wsdl/ver10/advancedsecurity/wsdl/advancedsecurity.wsdl

```diff
@@ -839,6 +875,127 @@
 				<xs:anyAttribute processContents="lax"/>
 			</xs:complexType>
 			<!--===============================-->
+			<xs:simpleType name="AuthorizationServerConfigurationType">
+				<xs:restriction base="xs:string">
+					<xs:enumeration value="OAuthAuthorizationCode">
+						<xs:annotation>
+							<xs:documentation>OAuth2 authorization code flow per RFC 6749</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="OAuthClientCredentials">
+						<xs:annotation>
+							<xs:documentation>OAuth2 client credentials grant flow per RFC 6749</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+				</xs:restriction>
+			</xs:simpleType>
+			<!--===============================-->
+			<xs:simpleType name="ClientAuthenticationMethod">
+				<xs:restriction base="xs:string">
+					<xs:enumeration value="client_secret_basic">
+						<xs:annotation>
+							<xs:documentation>Use HTTP Authorization header to specify client_secret par RFC 6749</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="client_secret_post">
+						<xs:annotation>
```

Comment:
suggestion to change "Use HTTP Authorization header to specify client_secret par RFC 6749" to "Use HTTP Authorization header to specify client_secret per RFC 6749". (i.e par ->per)

Reply:
Fixed